language/diem-framework/modules/CRSN.move

```diff
@@ -0,0 +1,310 @@
+/// A module implementing conflict-resistant sequence numbers (CRSNs).
+/// The specification, and formal description of the acceptance and rejection
+/// criteria, force expiration and window shifting of CRSNs is described in DIP-168.
+module 0x1::CRSN {
+    use 0x1::BitVector::{Self, BitVector};
+    use 0x1::Signer;
+    use 0x1::Errors;
+
+    friend 0x1::DiemAccount;
+
+    /// A CRSN  represents a finite slice or window of an "infinite" bitvector starting
```

Comment:
```suggestion
    /// A CRSN represents a finite slice or window of an "infinite" bitvector starting
```

Reply:
I think one example could help here; there were some great ones in the DIP/earlier docs. Also, could be helpful to say whether you're thinking of the infinite part as the left or right end.